packages/expo-modules-core/ios/AppDelegates/ExpoAppDelegateSubscriber.swift

```diff
@@ -23,7 +23,9 @@ open class BaseExpoAppDelegateSubscriber: UIResponder {
  Might be useful for compatibility reasons if we decide to add more things here.
  */
 @objc(EXAppDelegateSubscriberProtocol)
-public protocol ExpoAppDelegateSubscriberProtocol: UIApplicationDelegate {}
+public protocol ExpoAppDelegateSubscriberProtocol: UIApplicationDelegate {
+  @objc optional func customizeRootView(rctView: RCTRootView)
```

Comment:
```suggestion
  @objc optional func customizeRootView(rootView: UIView)
```

also UIView without sticking with RCTRootView